.github/workflows/nm-upload-assets-to-gcp.yml

```diff
@@ -68,6 +84,16 @@ jobs:
               with:
                   path: assets
 
+            - name: push wheel to pypi.org
```

Comment:
We also need to upload the .tar.gz file as well.

Reply:
+1

Reply:
Maybe we do something similar as .github/actions/nm-cp-assets/action.yml instead of passing the whl and tar.gz as input arguments.